docker/README.md

```diff
@@ -0,0 +1,29 @@
+# Creating docker containers for snapcraft
+
+## Setup
+
+    sudo snap install docker
+
+Ensure networking works, in some cases `dockerd` might need to have something
+like `--dns=8.8.8.8` added to the command.
```

Comment:
Can you put the command to ensure that networking works?
Something like docker run ... ping google.com ?

Reply:
This bit already existed, I just renamed the file for consistency. I added some bits at the bottom relating to this PR, but I'd rather not alter what is already there. Especially because I don't quite understand this blurb in the first place (why would it have DNS issues?).

Reply:
ok